server/server.go

```diff
@@ -119,6 +119,7 @@ func NewServer(ctx *Context, stopper *util.Stopper) (*Server, error) {
 	s.stopper.AddCloser(s.raftTransport)
 
 	s.kvDB = kv.NewDBServer(sender)
+	s.kvDB.RegisterRPC(s.rpc)
```

Comment:
We should put the server side of this behind a flag or something until we've made a more permanent decision about it.